CONTRIBUTING.md

```diff
@@ -34,12 +34,26 @@ If you haven't used `pipenv` before but are comfortable with virtualenvs, just r
 `pip install pipenv` in the virtualenv you're already using and invoke the command above
 from the cloned _Black_ repo. It will do the correct thing.
 
-Before submitting pull requests, run tests with:
+Before submitting pull requests, run lints, tests + CI with:
```

Comment:
CI is run only when pushing, right? Maybe leave it out from here?

```suggestion
Before submitting pull requests, run lints and tests with:
```

Reply:
Yeah I agree. I was just trying to say run Unit and Integration tests which you've achieved better. Thanks!